### PR TITLE
Proposal

### PR DESCRIPTION
**Pull request by brstanle**
- Added my sections & figures to the proposal.md file
- R Markdown for init data viz

### CODE REVIEW

File: reports/proposal.md [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+and an estimated $113 Billion USD in damages (Florida Department of Law Enforcement; National Centers for Environmental Information).
+Concerns were raised regarding the readability of the hurricane forecast maps and the "cone" used to predict the path of the hurricane (Dance and Ajasa).
+News outlets have suggested that the cone is misleading and that the public may not understand the information presented (Pulver and Doyle).
+This project aims to provide background to better understand how vision works and how to produce informative data visualizations.
+The project will be a stepping stone for continued research of data visualization and how to communicate hurricane forecast data effectively.
+</p>
+
+<p style="text-indent: 40px;">
+    Data for this project was collected in an experiment of 52 subjects enrolled in psychology 100 at CSU. Subjects were shown an image of dots via an online survey (see figure one) and 
+asked to select the quadrant where the centroid of the dots was located (see figure two); i.e., where is the middle of the dots? Each image was show to the subject for less than a second before the
```
Comment (murrayrodney) [inline reports/proposal.md]: Each image was shown, vs. show

File: reports/proposal.md [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
 ## Introduction
-* Context and background information
-* Study objectives
-* Experiment design
-* Other details
+
+<p style="text-indent: 40px;">
+    Hurricane Ian swept through the Hurricane Belt in 2022 leaving behind 149 deaths in Florida
+and an estimated $113 Billion USD in damages (Florida Department of Law Enforcement; National Centers for Environmental Information).
+Concerns were raised regarding the readability of the hurricane forecast maps and the "cone" used to predict the path of the hurricane (Dance and Ajasa).
```
Comment (murrayrodney) [inline reports/proposal.md]: Maybe reword to soemthing like:

the "cone" is used to show the predicted path...

File: reports/proposal.md [hunk 3]
```diff
@@ -1,8 +1,55 @@
...
+News outlets have suggested that the cone is misleading and that the public may not understand the information presented (Pulver and Doyle).
+This project aims to provide background to better understand how vision works and how to produce informative data visualizations.
+The project will be a stepping stone for continued research of data visualization and how to communicate hurricane forecast data effectively.
+</p>
+
+<p style="text-indent: 40px;">
+    Data for this project was collected in an experiment of 52 subjects enrolled in psychology 100 at CSU. Subjects were shown an image of dots via an online survey (see figure one) and 
+asked to select the quadrant where the centroid of the dots was located (see figure two); i.e., where is the middle of the dots? Each image was show to the subject for less than a second before the
+screen would change. Asking the subject to select the quadrant the centroid of the dots was in. The subjects' response was recorded along with metadata about the image. Each subject was
```
Comment (murrayrodney) [inline reports/proposal.md]: Maybe reword to:

After the image was shown for less than one second, the subject was asked to select which quadrant the centroid of the dots was in.

File: reports/proposal.md [hunk 4]
```diff
@@ -1,8 +1,55 @@
...
+  Figure 1: Experiment Visual Stimulus Example
+</p>
+
+![Figure 2: Experiment Quadrants](../references/figures/quad_options_example.png "Figure 2: Experiment Quadrants")
+
+<p align="center">
+  Figure 2: Experiment Centroid Quadrant Options
+</p>
+
+Explain the study design. Include details such as sample size, whether this is an observational
```
Comment (murrayrodney) [inline reports/proposal.md]: Can remove this section

File: reports/proposal.md [hunk 5]
```diff
@@ -1,8 +1,55 @@
...
+- `currImage`: the image ID (factor: 1-400)
+- `subjID`: the subject ID (factor: 1-52)
+- `resp`: subject response quadrant (factor: A, B, C, D)
+- `corrQuad`: the correct quadrant (factor: A, B, C, D)
+- `distToMiddle`: the distance from the centroid to the center of the screen (numeric)
+- `meanSD`: the standard deviation of all the dots (numeric)
+- `isColSame`: whether the quadrant for the mean of the red dots is the same quadrant as the mean of all the dots (boolean: 0, 1)
+- `isSizeSame`: whether the quadrant for the mean of the big dots is the same quadrant as the mean of all the dots (boolean: 0, 1)
+- `isPulseSame`: whether the quadrant for the mean of the dots with black outlines is the same quadrant as the mean of all the dots (boolean: 0, 1)
+- `howManyCorr`: how many of the special groups (red, big, black outlines) have their mean in the same quadrant as the quadrant for the full group of dots (factor: 0-3)
```
Comment (murrayrodney) [inline reports/proposal.md]: Ah, you'll see that I have a table that shows something similar, but also indicates which variables will be included as fixed vs. random effects. I don't think we need it twice, and maybe would be better in the proposed methods section?

**Review by murrayrodney — COMMENTED** [review]
Overall looks great, thanks for getting this together! For the most part I have only a couple of minor comments on spelling and wording changes. I did see we both included a summary of variables; I think you'll agree we only need to include one. In her requirements, it seems like it might belong in the methods section, so maybe we want to keep it there and we can make some changes to how I presented that? 

Go ahead and merge, once you feel that you've taken care of the pieces I commented on

File: reports/proposal.md (hunk 5 above)
Comment (brstanle) [reply reports/proposal.md]: I'll remove this!
